package.json

```
@@ -3,23 +3,32 @@
   "version": "2.0.2-0",
   "description": "An opinionated, isomorphic HTTP client.",
   "license": "BSD-3-Clause",
-  "main": "index.js",
+  "type": "module",
+  "main": "./dist/cjs/index.cjs",
+  "exports": {
+    "require": "./dist/cjs/index.cjs",
+    "import": "./lib/index.js"
+  },
   "scripts": {
+    "rollup": "rollup -c rollup.config.js",
+    "build": "npm run clear && npm run rollup",
+    "clear": "rimraf dist/ && mkdir dist",
+    "prepare": "npm run build",
+    "rebuild": "npm run clear && npm run build",
     "test": "npm run test-node",
-    "test-node": "cross-env NODE_ENV=test mocha -r esm --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
-    "test-karma": "karma start karma.conf.js",
-    "test-watch": "cross-env NODE_ENV=test mocha -r esm --watch --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
+    "test-node": "cross-env NODE_ENV=test mocha --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
+    "test-karma": "karma start karma.conf.cjs",
+    "test-watch": "cross-env NODE_ENV=test mocha --watch --parallel --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
     "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
     "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm run test-node",
     "coverage-report": "nyc report",
     "lint": "eslint ."
   },
   "files": [
-    "index.js",
-    "main.js"
+    "lib/*",
+    "dist/*"
   ],
   "dependencies": {
-    "esm": "^3.2.22",
     "ky": "^0.25.1",
     "ky-universal": "^0.8.2"
```

Update these to newer pure ESM releases:
```suggestion
    "ky": "^0.25.1",
    "ky-universal": "^0.8.2"
```

---

Updated [694880b](https://github.com/digitalbazaar/http-client/pull/21/commits/694880be30b62dfa587c2b1dfb1a00d05ac7e27a)

---

Also updated changelog [62d4dd1](https://github.com/digitalbazaar/http-client/pull/21/commits/62d4dd1b117a7b8fdeef96928f369a6cb3202d47)